static/gsApp/index.tsx

```diff
@@ -1,41 +0,0 @@
-import ConfigStore from 'sentry/stores/configStore';
-
-async function gsApp() {
-  try {
-    const [{bootstrap}, {initializeLocale}] = await Promise.all([
-      import('sentry/bootstrap'),
-      import('sentry/bootstrap/initializeLocale'),
-    ]);
-
-    const data = await bootstrap();
-
-    // This is called in `initializeMain` but we want to do this before `registerHooks` to
-    // avoid some warnings.
-    await initializeLocale(data);
-
-    // We have split up the imports this way so that locale is initialized as
-    // early as possible, (e.g. before `registerHooks` is imported otherwise the
-    // imports in `registerHooks` will not be in the correct locale.
-    const [{default: registerHooks}, {initializeBundleMetrics}, {initializeMain}] =
-      await Promise.all([
-        import('getsentry/registerHooks'),
-        import('getsentry/initializeBundleMetrics'),
-        import('sentry/bootstrap/initializeMain'),
-      ]);
-
-    // getsentry augments Sentry's application through a 'hook' mechanism. Sentry
-    // provides various hooks into parts of its application. Thus all getsentry
-    // functionality is initialized by registering its hook functions.
-    registerHooks();
-
-    await initializeMain(data);
-    initializeBundleMetrics();
-  } catch (err) {
-    window.Sentry?.captureException(err);
```

Comment:
@evanpurkhiser i feel like this isn't needed? though i feel we should be moving sentry init (in initializeApp) before initializeLocale

Reply:
initializeLocale needs to happen very early since we start using translations very early.

Reply:
oh, yeah maybe I see what you're saying. Actually idk.

i would avoid moving too much of that around right now if you can